app/mailers/form_submission_mailer.rb

```diff
@@ -28,4 +30,9 @@ def email_confirmation_input(text_input:, notify_response_id:, submission_email:
   def make_notify_boolean(bool)
     bool ? "yes" : "no"
   end
+
+  def link_to_csv_file(mailer_options, csv_file)
+    filename = "GOV_UK_Forms_Submission_#{mailer_options.title.parameterize(separator: '_')}_#{mailer_options.submission_reference}.csv"
```

Comment:
This line is getting a bit long and I found it a bit tricky to read at first glance - might be worth splitting out the processing from the interpolation:
```suggestion
    title = mailer_options.title.parameterize(separator: '_')
    submission_reference = mailer_options.submission_reference

    filename = "GOV_UK_Forms_Submission_#{title}_#{submission_reference}.csv"
```

Reply:
Yeh good point, have updated to extract this to a method and split out the interpolation